chain/cosmos/chain_node.go

```diff
@@ -191,7 +192,7 @@ func (tn *ChainNode) NewSidecarProcess(
 		VolumeName: v.Name,
 		ImageRef:   image.Ref(),
 		TestName:   tn.TestName,
-		UidGid:     image.UidGid,
+		UIDGID:     image.UIDGID,
```

Comment:
user breaking - lets until v10 - create ticket for this. Don't want people to have to refactor their test

Reply:
reverted in 0e64f2cfb0a1a6863740fdb0d72c2f649321d6cd